<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMSDlg</class>
 <widget class="QDialog" name="CMSDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>530</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PKCS#7 Cryptographic Message Syntax</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Output Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="mOutputBase64Radio">
        <property name="text">
         <string>Base64</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="mOutputHexRadio">
        <property name="text">
         <string>Hex</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="mOutputStringRadio">
        <property name="text">
         <string>String</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="4">
       <widget class="QPlainTextEdit" name="mOutputText"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>KM PrivateKey</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="mSignPriKeyPathText"/>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="mSignCertPathText"/>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Source Data</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="4">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QRadioButton" name="mSrcStringRadio">
        <property name="text">
         <string>String</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0" colspan="5">
       <widget class="QPlainTextEdit" name="mSrcText"/>
      </item>
      <item row="0" column="3">
       <widget class="QRadioButton" name="mSrcBase64Radio">
        <property name="text">
         <string>Base64</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QRadioButton" name="mSrcHexRadio">
        <property name="text">
         <string>Hex</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QPushButton" name="mChangeBtn">
     <property name="text">
      <string>Change</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>KM Certificate</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Sign Certificate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <layout class="QGridLayout" name="gridLayout_4">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item row="1" column="2">
      <widget class="QPushButton" name="mDevelopedAndVerifyBtn">
       <property name="text">
        <string>DevelopedAndVerify</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="mSignedDataBtn">
       <property name="text">
        <string>SignedData</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="mVerifyDataBtn">
       <property name="text">
        <string>VerifyData</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="mEnvelopedDataBtn">
       <property name="text">
        <string>EnvelopedData</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="mDevelopedDataBtn">
       <property name="text">
        <string>DevelopedData</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="mSignAndEnvelopedBtn">
       <property name="text">
        <string>SignAndEnveloped</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="mKMPriKeyFindBtn">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QPushButton" name="mSignCertFindBtn">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="mKMPriKeyPathText"/>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="mKMCertPathText"/>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="mSignPriKeyFindBtn">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="mKMCertFindBtn">
     <property name="text">
      <string>Find</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Sign PrivateKey</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QPushButton" name="mCloseBtn">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="mViewBtn">
     <property name="text">
      <string>View</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
